src/libp2p/discv5.ts

```diff
@@ -63,7 +67,7 @@ export class Discv5Discovery extends EventEmitter {
       config: options,
       metrics: options.metrics,
     });
-    this.searchInterval = options.searchInterval;
+    this.searchInterval = options.searchInterval ?? 0;
```

Comment:
Declare at the top of the file as `DEFAULT_SEARCH_INTERVAL_MS` and explain why this number and our roadmap to fine tune it